ui/src/components/app.vue

```diff
@@ -133,6 +146,23 @@ export default {
         if(this.app) this.app_ = this.app;
     },
 
+    computed: {
+        isIncompatible() {
```

Comment:
To simplify logic, prefer to use "positive" names and attributes.

For example, here `isIncompatible` is used later in `!this.isIncompatible`, which is a double negation. Is it not not compatible?

Better keep it as `isCompatible`